flowetl/tests/unit/test_csv_ingestion.py

```diff
@@ -0,0 +1,85 @@
+from flowetl.staging.staging import StagingManager, ExampleMappingStep, get_cursor
+import docker
+import tarfile
+from tempfile import NamedTemporaryFile
+from pathlib import Path
+
+
+import pytest
+
+
+@pytest.fixture
+def flowmachine_env(monkeypatch):
```

Comment:
Best handled outside the tests, no? pipenv does this itself, or one can use pytest-dotenv or similar?

Reply:
We can drop this entire file, actually - this is a holdover from before Airflow that I forgot to drop.